packages/server/src/cli.ts

```diff
@@ -166,6 +166,20 @@ if (isCli) {
 		password = buffer.toString("hex");
 	}
 
+	// If CLI password was provided, obsfucate password from process title
+	if (flags.password) {
+		const parts = [process.title];
+		parts.push(path.relative(process.cwd(), process.argv[1]));
+		for (let i = 2; i < process.argv.length; i++) {
+			if (process.argv[i].includes("--password")) {
+				parts.push(process.argv[i].replace(/=(.*)/, "=****"));
+			} else {
+				parts.push(process.argv[i]);
+			}
+		}
+		process.title = parts.join().replace(/\,/g, " ");
```

Comment:
This will look bad if any of the parameters have a comma in it. It's also sub-optimal. Why not provide a parameter for the `join()` function?

```suggestion
		process.title = parts.join(" ");
```

Reply:
Good call!